integrations/pjrt/python_packages/iree_cuda_plugin/setup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python3
+
+# Copyright 2023 The OpenXLA Authors
```

Comment:
some of these files are "The OpenXLA Authors" - is that true/something true going forward?

Reply:
I've been slapped enough that I don't touch copyrights. Someone from the google side is legally able to change this and can do it in a followup.

@jpienaar